Insertion in Linked List

YashikaAggarwal opened the pull request:
<!--
- Add the issue number here(after hashtag below adjacent to FIXES).

- If you haven't solved the issue completely use "Linked issue #{issue_number}.
- After solving the issue completely change it to "Fixes #{issue_number}.
-->
Fixes #

<!-- Add here what changes were made in this issue and if possible provide links. -->

YashikaAggarwal commented:
Hi, Please review my Pull Request

Akshat05-tech commented:
@YashikaAggarwal please add relevant comments in your code for understandable to all. Also mention the issue number you solved after Fixes#

s-ayush2903 commented:
@YashikaAggarwal What's the status here?

YashikaAggarwal commented:
> @YashikaAggarwal What's the status here?

I have added some comments.
Also, I want to inform that I wrongly mentioned the name of program as Insertion in Linked List. It is actually program for finding no. of binary trees of height h. I have updated the same. I am really sorry for the confusion caused.

YashikaAggarwal commented:
> > @YashikaAggarwal What's the status here?
> 
> I have added some comments.
> Also, I want to inform that I wrongly mentioned the name of program as Insertion in Linked List. It is actually program for finding no. of binary trees of height h. I have updated the same. I am really sorry for the confusion caused.

Please review it. Thank You :)
